baseband/mark5b/base.py

```diff
@@ -291,26 +302,40 @@ class Mark5BStreamWriter(VLBIStreamWriterBase, Mark5BFileWriter):
     _frame_class = Mark5BFrame
 
     def __init__(self, raw, frames_per_second=None, sample_rate=None,
-                 nchan=1, bps=2, header=None, **kwargs):
+                 nchan=1, bps=2, header=None, squeeze=True, **kwargs):
         if header is None:
             header = Mark5BHeader.fromvalues(**kwargs)
+        sample_shape = Mark5BPayload._sample_shape_cls(nchan)
```

Comment:
Seeing this time and again, I wonder whether `vlbi_base/base.py` should simply use `self._payload_cls._sample_shape_cls(...)` but let's leave that for possible follow up.